pydata_sphinx_theme/bootstrap_html_translator.py

```diff
@@ -37,7 +37,10 @@ def visit_table(self, node):
         if LooseVersion(sphinx.__version__) < LooseVersion("4.0"):
             self.generate_targets_for_table(node)
 
-        self._table_row_index = 0
+        if LooseVersion(sphinx.__version__) < LooseVersion("4.3"):
+            self._table_row_index = 0
+        else:
+            self._table_row_indices.append(0)
```

Comment:
The second and subsequent elements of the list are used for the nested tables. So you should not append the value if you'd like to make a new nested table.

The last element always points to the current table. So it's better to modify its value like following:
```
            self._table_row_indices[-1] = 0
```